### DISCUSSION TITLE
A redundant () when set degree-type=professional?

### DISCUSSION THREAD

**Discussion by gengzll** [ustctug/ustcthesis#322] [Q&A]
![image](https://user-images.githubusercontent.com/36037808/146120612-a3a0a8b5-c8b6-41ed-ba27-bc21742c6540.png)

**Comment by sikouhjw** [comment]
This is not superfluous. You need to type `professional-type`.
https://github.com/ustctug/ustcthesis/blob/deb14d4da9dce890ff50a97db823fec6b2d0c61c/ustcsetup.tex#L14-L15

**Reply by zepinglee** [reply]
Thanks for helping answering.

**Comment by zepinglee** [comment]
I'll add some hints in the PDF output when `degree-type=professional` and `professional-type` is left empty (b3f73ff).
